tests/e2e/volumeproperty_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is an end-to-end test. Test will ssh to a vm and create a volume using docker cli.
+// After creating a volume, test will ssh to the ESX and verify the properties of the volume.
+// Properties being verified - capacity, disk-format and vm-attached field.
+
+// Test assumes that SSH cert has been setup to enable password-less login to VM and ESX.
+
+package e2e
+
+import (
+	"log"
+	"strings"
+
+	dockerconst "github.com/vmware/docker-volume-vsphere/tests/constants/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/inputparams"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/verification"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	TestName           = "volumeproperty"
+	DiskStatusDetached = "detached"
+	DiskStatusEmpty    = "<novalue>"
+)
+
+type VolumePropVerificationSuite struct {
+	volumeNames           [3]string
+	dockerHostIP          string
+	dockerHostName        string
+	containerName         string
+	esxIP                 string
+	formatTypes           []string
+	dockerVolumeRemoveCmd string
```

Comment:
I am creating a single command to remove all the volumes so that we don't need to ssh for each volume.
This is how final command looks like: docker volume rm vol1 vol2 vol3